ci-bootstrap/README.md

```diff
@@ -0,0 +1,7 @@
+Resources for setting up roles in a new AWS account
+
```

Comment:
Some more explanation here would be good as well to explain exactly what this bit is for and how/when to run it.

Reply:
I've gone with this (https://github.com/ONSdigital/statement-of-record-terraform/blob/master/ci-bootstrap/README.md) for statement of record.

Also referenced it in the main README

Reply:
Added changes to the Readme file to explain the functionality better